static/app/views/settings/account/apiNewToken.tsx

```diff
@@ -31,19 +34,22 @@ export default class ApiNewToken extends Component<{}, State> {
         Release: 'no-access',
         Organization: 'no-access',
       },
+      newToken: null,
     };
   }
 
   onCancel = () => {
     browserHistory.push(normalizeUrl(API_INDEX_ROUTE));
   };
 
-  onSubmitSuccess = () => {
+  handleGoBack = () => {
     browserHistory.push(normalizeUrl(API_INDEX_ROUTE));
   };
 
   render() {
     const {permissions} = this.state;
```

Comment:
you can do this in one assignment: 
```suggestion
    const {permissions, newToken} = this.state;
```